index.html

```diff
@@ -127,42 +127,70 @@
                   </nav>
                 </div>
 
-                <div class="text sub-link">
+                <div class="text ">
                   <div id="text-d101b3773d" class="cmp-text">
-                    <p><a href="#">Leaderhub</a> <a href="#">Give</a> <a href="#">Contact</a></p>
+                    <div class="sub-link"><a class="sub-link-item" href="#">Leaderhub</a> <a class="sub-link-item" href="#">Give</a> <a class="sub-link-item" href="#">Contact</a></div>
                   </div>
                 </div>
 
                 <div class="container responsivegrid mobile-nav-footer">
-                  <div id="container-30767cf08c" class="cmp-container">
+                  <div id="nav-bottom-footer" class="cmp-container">
                     <div class="container responsivegrid term-policy-wrap">
                       <div id="container-9bd027d484" class="cmp-container">
                         <div class="text">
                           <div id="text-b4e9c1f140" class="cmp-text">
-                            <p><a href="#">Privacy</a> <a href="#">Terms of Use</a></p>
+                            <div class="terms-grid">
+                              <span class="term-policy-item" data-cmp-data-layer="">
+                              <a data-cmp-clickable="" class="nav-link-item" href="">Privacy</a>
+                          </span>
+                              <span class="term-policy-item" data-cmp-data-layer="">
+                              <a data-cmp-clickable="" class="nav-link-item" href="">Terms of Use</a>
+                          </span>
+                           </div>
                           </div>
                         </div>
                       </div>
                     </div>

```

Comment:
This HTML looks like this:
```
<div class="container responsivegrid term-policy-wrap">
  <div id="container-aa1adff3d8" class="cmp-container">
    <div class="container responsivegrid terms-grid">
      <div id="container-10bae5eac8" class="cmp-container">
        <div class="text term-policy-item">
          <div id="text-5b59f11baa" class="cmp-text">
            <p><a href="#">Privacy</a></p>
          </div>
        </div>
        <div class="text term-policy-item">
          <div id="text-703f7fc464" class="cmp-text">
            <p><a href="#">Terms of Use</a></p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
```

Or we could use buttons here instead. Or another list like the above comment.